src/main/kotlin/uk/gov/justice/digital/hmpps/approvedpremisesapi/service/UserAccessService.kt

```diff
@@ -87,6 +87,18 @@ class UserAccessService(
     else -> false
   }
 
+  /**
+   * Currently delegates to the lost bed logic. May depend on different roles in the future.
+   */
+  fun currentUserCanManagePremisesOutOfServiceBed(premises: PremisesEntity) =
+    userCanManagePremisesOutOfServiceBed(userService.getUserForRequest(), premises)
+
+  /**
+   * Currently delegates to the lost bed logic. May depend on different roles in the future.
+   */
+  fun userCanManagePremisesOutOfServiceBed(user: UserEntity, premises: PremisesEntity) =
+    userCanManagePremisesLostBeds(user, premises)
```

Comment:
yes this works well, we can follow up with PRs to make:

- out-of-service beds scoped by premises available to `FUTURE_MANAGER`  with a new `UserAccessService` function such as`userCanManagePremisesOutOfServiceBeds(user, premises)`
- make out-of-service beds (across whole estate) available to `WORKFLOW_MANAGER` or (similar role for CRU staff) with a new `UserAccessService` function such as`userCanViewAllOutOfServiceBeds(user)`